_posts/README.md

```diff
@@ -0,0 +1,13 @@
+# Auto-convert markdown files To Posts
+
+[`fastpages`](https://github.com/fastai/fastpages) will automatically convert markdown files saved into this directory as blog posts!
+
+You must save your notebook with the naming convention `YYYY-MM-DD-*.md`.  Examples of valid filenames are:
+
+```shell
+2020-01-28-My-First-Post.md
+2012-09-12-how-to-write-a-blog.md
+```
+
+See [Writing Blog Posts With Jupyter](https://github.com/saketkc/fastpages#writing-blog-posts-with-markdown) for more details.
```

Comment:
This links back to @saketkc's fork, and not a README file. Looks like you've meant to link to the README in root folder? If so, a `[Writing Blog Posts With Jupyter](../README.md#writing-blog-posts-with-markdown)` would do the trick.

FYI @hamelsmu

Reply:
Wow I didn’t catch that

Reply:
I'll correct this now

Reply:
Sorry, that was an overlook on my side.